lib/rules/media-feature-name-value-no-unknown/__tests__/index.js

```diff
@@ -0,0 +1,196 @@
+'use strict';
+
+const { messages, ruleName } = require('..');
+
+testRule({
+	ruleName,
+	config: [true],
+
+	accept: [
```

Comment:
major

Could you add a test for `screen-spanning`?

browser: edge
see: https://daverupert.com/2020/07/gettin-foldy-with-the-dual-screen-web-part-i/

Reply:
`screen-spanning` is an unknown media feature and those are ignored by this rule.

`media-feature-name-no-unknown` already exists to cover those issues.